modules/mono/glue/Managed/Files/Color.cs

```diff
@@ -114,6 +264,11 @@ public Color Black
             }
         }
 
+        public static Color ColorN(string name)
```

Comment:
This should take a second parameter for the alpha: http://docs.godotengine.org/en/latest/classes/class_@gdscript.html#class-gdscript-colorn